src/generators/bendable-amogus/BendableAmogus.res

```diff
@@ -0,0 +1,239 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
+
+let name = "Bendable Amogus generator"
+
+let images: array<Generator.imageDef> = [
+  {id: "Background", url: requireImage("Background.png")},
+  {id: "Folds", url: requireImage("Folds.png")},
+  
+]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin",
+    url: requireTexture("Skin.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Colors",
+    url: requireTexture("Colors.png"),
+    standardWidth: 16,
+    standardHeight: 2,
+  },
+]
+
+let script = () => {
+  Generator.defineSelectInput("Color", [
+        "Red",
+        "Black",
+        "White",
+        "Rose",
+        "Blue",
+        "Cyan",
+        "Yellow",
+        "Pink",
+        "Purple",
+        "Orange",
+        "Banana",
+        "Coral",
+        "Lime",
+        "Green",
+        "Gray",
+        "Maroon",
+        "Brown",
+        "Tan",
+        
+      ]);
+
+      Generator.defineTextureInput("Skin", {
+        standardWidth: 64,
+        standardHeight: 64,
+        choices: []
+      });
+      Generator.drawImage("Background", (0, 0))
+  let red = Generator.getSelectInputValue("Color") === "Red";
```

Comment:
There are a few formatting problems in the file.

If you use VSCode for editing the files then install the `rescript-vscode` extension. This should auto-format the file each time you save the file.

Reply:
I already have it installed, but it does not changes anything. How can I fix it?

Reply:
Done